awx/conf/license.py

```diff
@@ -1,19 +0,0 @@
-# Copyright (c) 2016 Ansible, Inc.
-# All Rights Reserved.
-
-# Tower
-from awx.main.utils.common import get_licenser
-
-__all__ = ['get_license']
-
-
-def _get_validated_license_data():
-    return get_licenser().validate()
```

Comment:
...here?

Reply:
That worked, thanks for the suggestion.  Currently I get a 400 back, `File or Upload field not found`.  My guess is when setting the headers field I am overwriting whatever file headers the requests library adds when I specify `files=<tar-path>`.